util/fipstools/delocate/delocate.go

```diff
@@ -59,6 +59,30 @@ const (
 	aarch64
 )
 
+// represents a unique symbol for an occurrence of OPENSSL_ia32cap_P.
+type cpuCapUniqueSymbol struct {
+	registerName string
+	suffixUniqueness string
+}
+
+const cpuCapIntelSymbolPrefix = "LOPENSSL_ia32cap_P_"
+
+func (uniqueSymbol cpuCapUniqueSymbol) getIntelSymbol() string {
+	return cpuCapIntelSymbolPrefix + uniqueSymbol.registerName + uniqueSymbol.suffixUniqueness
+}
+
+func (uniqueSymbol cpuCapUniqueSymbol) getIntelSymbolReturn() string {
+	return uniqueSymbol.getIntelSymbol() + "_return"
+}
+
+// uniqueness must be a globally unique integer value.
+func newCpuCapUniqueSymbol(uniqueness int, registerName string) *cpuCapUniqueSymbol {
+    uniqueSymbol := new(cpuCapUniqueSymbol)
```

Comment:
Simplify to:
```go
return &uniqueSymbol{
   registerName: strings.Trim(registerName, "%"), // should work with both AT&T and Intel syntax.
   suffixUniqueness: strconv.Itoa(uniqueness),
}
```
Which is a bit more idiomatic

Reply:
great!